lib/manageiq/providers/workflows/builtin_methods.rb

```diff
@@ -14,9 +14,27 @@ def self.email(params = {}, _secrets = {}, _context = {})
           miq_task_status!(runner_context)
         end
 
+        def self.provision_execute(_params = {}, _secrets = {}, context = {})
+          # ensure we are in a provisioning request
+          unless context.execution["_object_type"] == "ServiceTemplateProvisionTask"
+            return BuiltinRunner.error!(runner_context, :cause => "Calling provision_execute on non-provisioning request: [#{runner_context["_object_type"]}]")
+          end
+
+          request_task_id = context.execution["_object_id"]&.to_i
+          ::MiqRequestTask.find(request_task_id)&.execute
```

Comment:
@kbrock it looks like we go through some effort to ensure `context.execution["_object_type"]` is `"ServiceTemplateProvisionTask"` but then also go through a lot of effort to handle a missing `_object_id`.

Should we also be raising an error if `context.execution["_object_type"]` is missing and if `::MiqRequestTask.find(request_task_id)` can't be found?